tap-agent/src/agent/sender_accounts_manager.rs

```diff
@@ -157,7 +152,14 @@ impl Actor for SenderAccountsManager {
                 .await?;
         }
 
+        state.new_receipts_watcher_handle = Some(tokio::spawn(new_receipts_watcher(
+            pglistener,
+            escrow_accounts,
+            prefix,
+        )));
+
         tracing::info!("SenderAccountManager created!");
+        // start database monitor
```

Comment:
I think this comment should be removed. Or maybe I don't understand what it means here.